packages/editor/src/components/post-title/index.js

```diff
@@ -141,7 +141,7 @@ const applyWithSelect = withSelect( ( select ) => {
 
 	return {
 		isCleanNewPost: isCleanNewPost(),
-		title: getEditedPostAttribute( 'title' ),
+		title: getEditedPostAttribute( 'title' ) || ownTitle,
```

Comment:
With this priority, you will never be able to use the override on a page that already has a title assigned and `getEditedPostAttribute` will be returning value. It will only work on title-less page which I don't think is the goal here

Reply:
I thought `title` will try to get the value from `getEditedPostAttribute( 'title' )` as first option. If it isn't defined then it will try to apply the second part which is `ownTitle`. Testing again.

Reply:
> It will only work on title-less page which I don't think is the goal here

Actually it's the desired goal here

Reply:
do you think that the explicit value should be stronger than site post?

Reply:
I think it depends on what you are trying to achieve.

If the custom title provided as an attribute is conceptually a "placeholder" value or not.

In my head the data flow model suggests that if you provide a prop such as `title` then you should expect that title to be reflected in the rendered component. To have a side effect suddenly change the title to something else entirely could feel very odd. I've provided a `title` so show that title.

Alternatively if you call the prop `fallbackTitle` then it might be legitimate to have the title rendered unless the post attribute title isn't available.

Reply:
What do you think about `__experimentalFallbackTitle_`?

Reply:
This way you can really only use the component override in an iframe (where there will never be a post title) but that's not how normal usages would look like. If we make it `title` and change the order in the condition, I think that would be a much more useful API for others too. That way you will be able to render strings styled as titles anywhere. Up to you to make the decision but I find it strange in the current form

Reply:
yeah, I think makes sense what you do suggest.[ Addressed here](8fccebbdc2433fca794bba28f28ed279d2f15b78).